hr_holidays_accrual_advanced/models/hr_leave_allocation.py

```diff
@@ -240,6 +243,18 @@ def _update_accrual(self):
         for allocation in allocations:
             allocation._update_accrual_allocation()
 
+        holidays = self.search([
+            ('accrual', '=', True), ('employee_id.active', '=', True),
+            ('state', '=', 'validate'), ('holiday_type', '=', 'employee'),
+            ('accrual_method', 'in', ('start_of_month', 'end_of_month')), '|', ('date_to', '=', False),
+            ('date_to', '>', fields.Datetime.now()), ('nextcall', '!=', False)])
+        for holiday in holidays:
+            if holiday.accrual_method == 'start_of_month':
```

Comment:
I'd do that differently, since relying on `nextcall` is not how this module works, see https://github.com/OCA/hr/blob/12.0/hr_holidays_accrual_advanced/models/hr_leave_allocation.py#L660

Your current approach won't show historical accrual data over time

Reply:
Sorry @alexey-pelykh 

I do not understand this, could you show me how to make this, please?

I do not understand how you get the nextcall.

Regards

Reply:
`hr_holidays_accrual_advanced` does not rely on nextcall at all, it completely overrides the accrual holidays calculation from Odoo, adopting accruement approaches from more mature HRIS packages.